skunkworks_crow/src/main/java/org/odk/share/views/ui/bluetooth/BluetoothListAdapter.java

```diff
@@ -49,7 +50,9 @@ public void setDevices(List<BluetoothDevice> dateSet) {
 
     public void addDevice(BluetoothDevice bluetoothDevice) {
         if (!bluetoothDeviceList.contains(bluetoothDevice)) {
-            this.bluetoothDeviceList.add(bluetoothDevice);
+            if (bluetoothDevice.getBluetoothClass().getDeviceClass() == BluetoothClass.Device.PHONE_SMART) {
```

Comment:
This can result into NPE, please handle those cases.

Reply:
ok, sure @lakshyagupta21 
I don't have the tablet, one of my friend have it I send the APK to him and I will let you know when I got a response from him @huangyz0918

Reply:
> have you test for the android tablets? Is the change disable the tablets?

@huangyz0918 My friend checked and he said it is showing as expected
@lakshyagupta21 I added the check for NPE, can you please review now